### PR TITLE
commit #1

### PR DESCRIPTION
**Pull request by la2798**
~added logo and progress bar
changed the main_activity and made it as a login page
~added button and image

### CODE REVIEW

**Comment by piedcipher** [comment]
Gradle build is failing. Please add missing drawables in activity_splash.xml and activity_main.xml

**Comment by la2798** [comment]
All the files required are there in the drawable, the gradle had not synced properly , how shall i update the PR?

**Comment by la2798** [comment]
I updated , can u please let me know if the error is gone?

File: app/src/main/res/layout/activity_splash.xml [hunk 1]
```diff
@@ -1,9 +1,29 @@
...
-</android.support.constraint.ConstraintLayout>
\ No newline at end of file
+    <ImageView
+        android:id="@+id/imageView"
+        android:layout_width="wrap_content"
+        android:layout_height="wrap_content"
+        android:layout_alignParentStart="true"
+        android:layout_alignParentTop="true"
+        android:layout_marginStart="21dp"
+        android:src="@drawable/wmnsafety" />
```
[the remark this answers is not in the document]
Comment (piedcipher) [reply app/src/main/res/layout/activity_splash.xml]: No it isn't. Push them to fork & modify PR.

**Comment by piedcipher** [comment]
Welldone! Great work @la2798. :+1: :hearts: Thank you for contributing.

**Comment by piedcipher** [comment]
Thank you for contributing :+1:
Merged #27.

**Pull request merged**